apps/main/src/llamalend/features/bands-chart/TooltipContent.tsx

```diff
@@ -0,0 +1,82 @@
+import { Token } from '@/llamalend/features/borrow/types'
+import { TooltipItem, TooltipItems, TooltipWrapper } from '@/llamalend/widgets/tooltips/TooltipComponents'
+import { Box, Typography } from '@mui/material'
+import { t } from '@ui-kit/lib/i18n'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { formatNumber, formatPercent, formatUsd } from '@ui-kit/utils'
+import { ChartDataPoint } from './types'
+
+const { Spacing } = SizesAndSpaces
+
+type TooltipContentProps = {
+  data: ChartDataPoint
+  collateralToken: Token | undefined
+  borrowToken: Token | undefined
+}
+
+const calculateBandShare = (numerator: number | undefined, denominator: number | undefined): string =>
+  typeof denominator === 'number' && denominator > 0 && typeof numerator === 'number'
+    ? `${formatPercent((numerator / denominator) * 100)}`
+    : '?'
+
+const formatAbbreviatedNumber = (value: number | undefined): string =>
+  typeof value === 'number' ? `${formatNumber(value, { abbreviate: true })}` : '?'
+
+export const TooltipContent = ({ data, collateralToken, borrowToken }: TooltipContentProps) => {
+  const hasMarketData =
+    (typeof data.bandCollateralAmount === 'number' && data.bandCollateralAmount > 0) ||
+    (typeof data.bandBorrowedAmount === 'number' && data.bandBorrowedAmount > 0)
+  const hasUserData = !!(data.userBandCollateralValueUsd || data.userBandBorrowedValueUsd)
+  const collateralBandShare = calculateBandShare(data.bandCollateralValueUsd, data.bandTotalCollateralValueUsd)
+  const borrowedBandShare = calculateBandShare(data.bandBorrowedValueUsd, data.bandTotalCollateralValueUsd)
+  const userBandShare = calculateBandShare(data.userBandTotalCollateralValueUsd, data.bandTotalCollateralValueUsd)
+
+  return (
+    <Box sx={{ padding: Spacing.md, backgroundColor: (t) => t.design.Layer[3].Fill }}>
+      <TooltipWrapper>
+        <Typography variant="bodyMBold" color="textPrimary">
+          {t`LLAMMA Band`}
+        </Typography>
+        <Typography variant="bodySRegular">{t`Active price range where collateral and debt rebalance.`}</Typography>
+        {hasUserData && (
+          <TooltipItems secondary>
+            <TooltipItem title={t`Your share of band`}>{userBandShare}</TooltipItem>
+            <TooltipItem variant="subItem" title={collateralToken?.symbol}>
+              {formatAbbreviatedNumber(data.userBandCollateralAmount)}
+              {`${formatUsd(data.userBandCollateralValueUsd ?? 0)}`}
```

Comment:
Showing $0 just because we don't have the token price seems incorrect.
Also, the template string is not needed here
```suggestion
              {formatUsd(data.userBandCollateralValueUsd ?? 0)}
```